Kerbee/Functions/RenewKeyActivity.cs

```diff
@@ -26,7 +26,7 @@ public async Task<object> RunAsync([ActivityTrigger] Application application)
         }
         catch (Exception ex)
         {
-            _logger.LogError(ex, "Error renewing key for application {ApplicationId}", application.Id);
```

Comment:
Wat is het verschil tussen `application.Id` en `application.AppId`? Voor de lezer zijn die property namen niet handig gekozen vind ik

Reply:
Ik zie dat `Id` staat voor `ObjectId`, maar wat is dat dan?

Reply:
Ieder object in AAD heeft een ObjectId, een app registrtion heeft daarnaast ook een AppId. Geen idee waarom dat 2 properties zijn, dat heeft Microsoft bedacht.